index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
Seems like these spaces are being used for readability, this is fine since it doesn't have any functionality. However, you could be on a team that might not prefer so be on the look out for that. There are tools that you could install to help with you keep track of things!